src/unlimited.cpp

```diff
@@ -1814,8 +1829,22 @@ UniValue getstat(const UniValue &params, bool fHelp)
         }
         else
         {
+            //Get time before request
+            int64_t SeriesStartTime = GetTimeMillis();
             UniValue series = base->GetSeries(seriesStr, count);
+            int64_t SeriesEndTime = GetTimeMillis();
```

Comment:
And I will look at how to present the series time from the perspective of the stat class.